manifests/observability-hub/overlays/openshift/grafana/dashboards.yaml

```diff
@@ -1,6 +1,8 @@
 apiVersion: grafana.integreatly.org/v1beta1
 kind: GrafanaDashboard
 metadata:
+  annotations:
+    argocd.argoproj.io/sync-options: SkipDryRunOnMissingResource=true
```

Comment:
This is required if the grafana crds are not present in the cluster. Otherwise argocd performs a check, detects that the API doesn't exists and immediately fails